src/styles/admin.css

```
@@ -1859,6 +1859,19 @@
   clip: rect(0 0 0 0);
 }
 
+.fuma-campaign-thumbnail-upload > input[type="url"] {
+  position: static;
+  width: 100%;
+  height: auto;
+  overflow: visible;
+  clip: auto;
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)